pkg/utils/logutil/log_test.go

```diff
@@ -33,99 +35,158 @@ func TestStringToZapLogLevel(t *testing.T) {
 	re.Equal(zapcore.InfoLevel, StringToZapLogLevel("whatever"))
 }
 
+func TestRedactInfoLogType(t *testing.T) {
+	re := require.New(t)
+	// JSON unmarshal.
+	jsonUnmarshalTestCases := []struct {
+		jsonStr string
+		expect  RedactInfoLogType
+	}{
+		{`false`, RedactInfoLogOFF},
+		{`true`, RedactInfoLogON},
+		{`"MARK"`, RedactInfoLogMark},
+		{`"OTHER"`, RedactInfoLogON},
```

Comment:
Why `OTHER` should be ON?

Reply:
I consider that if the user uses a string other than true/false and it is not `MARK`, at least the enabled state should be retained. Of course, this can also be discussed, and I remain open to this.

Reply:
Just reach an agreement with PM~

Reply:
I modified the behavior to be consistent with TiDB, only allowing it to be true/false/"MARKER". PTAL.